src/api/spec/controllers/webui/patchinfo_controller_spec.rb

```diff
@@ -290,5 +316,28 @@ def do_proper_post_save
       it { expect(JSON.parse(response.body)).to eq('error' => "hell is not a valid tracker.\n", 'issues' => []) }
       it { expect(response).to have_http_status(:success) }
     end
+
+    context 'if cve issue are wrong formatted' do
+      before do
+        get :new_tracker, params: { project: user.home_project_name, package: patchinfo_package.name, issues: ['CVE-2017-31337ABC'] }
+      end
+
+      it {
+        error_message = 'cve  has no valid format. (Correct formats are e.g. boo#123456, CVE-1234-5678 and the string has to be a comma-separated list)'
+        expect(JSON.parse(response.body)).to eq('error' =>
+                                                error_message,
+                                                'issues' => [])
+      }
+      it { expect(response).to have_http_status(:success) }
+    end
+  end
+
+  describe 'GET #delete_dialog' do
+    before do
+      login user
+      get :delete_dialog, xhr: true, params: { project: user.home_project_name, package: patchinfo_package.name }
+    end
+
+    it { expect(response).to have_http_status(:success) }
```

Comment:
I would test the delete dialog / patchinfo deletion in a feature test.Considering the [action's code](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/patchinfo_controller.rb#L177) this doesn't seem to test much right now:-)

Reply:
It was just to improve the test coverage

Reply:
The point of improving the test coverage is to make sure that we test everything (or more realistically as much as possible). Just touching LoC isn't enough:-)

Reply:
it was fixed implemented the patchinfo feature @bgeuken